pkg/tempopb/tempo.proto

```diff
@@ -80,6 +80,14 @@ message SearchBlockRequest {
   string version = 9;
   uint64 size = 10; // total size of data file
   uint32 footerSize = 11; // size of file footer (parquet)
+  repeated DedicatedColumn dedicatedColumns = 12;
+}
+
+// Configuration for a single dedicated attribute column.
+message DedicatedColumn {
+  string type = 1;
+  string name = 2;
+  string scope = 3;
```

Comment:
Will we want enums for `type` and `scope` eventually?

Reply:
So far we are using string values to represent `type` and `scope`. To me this makes sense as these are exposed to the user in the configuration and overrides.
AFAIK protocol buffers does not support enums with string values. This means we would have to convert back and forth. Not sure if this is worth the effort. What do you think?

Reply:
I added enums for type and scope to tempo.proto :+1: